engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreHeartbeatWriteActor.scala

```diff
@@ -22,13 +22,13 @@ case class WorkflowStoreHeartbeatWriteActor(workflowStoreAccess: WorkflowStoreAc
                                             terminator: CromwellTerminator,
                                             override val serviceRegistryActor: ActorRef)
 
-  extends EnhancedBatchActor[(WorkflowId, OffsetDateTime)](
+  extends EnhancedBatchActor[WorkflowStoreWriteHeartbeatCommand](
```

Comment:
I expected to see a corresponding change to the creation of these messages.

Reply:
The impact of this is mostly seen in the simplifications to the `override` signatures below. If I had continued with the existing case class to tuple conversion pattern I would have needed to add another tuple element so that would only have gotten worse, not that I saw the value of unpacking the case class params in the first place 🤷